### PR TITLE
RTEMIS-143: Campaign listing page.

### PR DESCRIPTION
**Pull request by RohanSingh-inno**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-143

---
### What the PR does
`(Explain fixes, changes and impacts)`
- Created a listing page for all the campaigns.

---
### What I have tested
- The campaign listing page and the campaign details are visible to the state admin.

---
### Extras
image

### CODE REVIEW

**Review by kunal-sin — CHANGES_REQUESTED** [review]
1. Please add the edit button as well for individual academic session.

File: modules/rte_mis_state/config/install/views.view.academic_sessions_list.yml [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+          group_rows: true
+          delta_limit: 0
+          delta_offset: 0
+          delta_reversed: false
+          delta_first_last: false
+          multi_type: separator
+          separator: ', '
+          field_api_classes: false
+      pager:
+        type: mini
```
[the remark this answers is not in the document]
Comment (RohanSingh-inno) [reply modules/rte_mis_state/config/install/views.view.academic_sessions_list.yml]: ok

**Comment by RohanSingh-inno** [comment]
> 1. Please add the edit button as well for individual academic session.

Ok.

File: config/install/menu_export.export_data.yml [hunk 2]
```diff
@@ -4404,6 +4513,54 @@
...
+  menu_name:
+    value: state-admin-menu
+  link:
+    uri: 'internal:/academic-sessions'
+    title: ''
+    options:
+      fa_icon: ''
+      fa_icon_prefix: fa
+      fa_icon_tag: i
+      fa_icon_appearance: before
```
[the remark this answers is not in the document]
Comment (RohanSingh-inno) [reply config/install/menu_export.export_data.yml:4447]: ok

File: config/install/menu_export.export_data.yml [hunk 3]
```diff
@@ -4408,13 +4418,110 @@
...
+    value: state-admin-menu
+  link:
+    uri: 'route:<nolink>'
+    title: ''
+    options:
+      fa_icon: ''
+      fa_icon_prefix: fa
+      fa_icon_tag: i
+      fa_icon_appearance: before
+  external:
```
Comment (kunal-sin) [inline config/install/menu_export.export_data.yml:4496]: Why we are adding the new link here? We already have a menu item. 

Screenshot from 2024-05-07 14-34-32

Comment (RohanSingh-inno) [reply config/install/menu_export.export_data.yml]: Ok so we will be following this. I have added a action link to create a academic session from the listing page itself.

**Review by kunal-sin — APPROVED** [review]

**Pull request merged**